README.md

```diff
@@ -1,44 +1,251 @@
+
+- You need to [register a new domain](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/domain-register.html) / [host a registered domain](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/MigratingDNS.html) you will use to access your infrastructure on [Amazon Route 53](https://aws.amazon.com/route53/).
+
+  - We recommended you have two domains one for staging another for production.
+
+___
+
+## Using zero to spin up your own stack
+
+Using Zero to spin up your infrastructure is easy and straightforward; using a few commands, you can configure and deploy your very own scalable high-performant infrastructure that is production-ready.
+
+A few caveats before getting started:
+
+- It is recomended practice to [create a GitHub org](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/creating-a-new-organization-from-scratch) where your code is going to live and grant [CircleCi Organization access](https://github.com/settings/connections/applications/78a2ba87f071c28e65bb) to your repositories for CI / CD deployment.
+
+- During set-up, you need to create a GitHub personal access token and a CircleCi access token; you should store your generated tokens securely.
+
+- For your CI build to work, you need to opt into the use of third-party orbs you can find this in your CircleCi Org Setting > Security > Allow Uncertified Orbs.
+
+- It takes about 20 - 35 mins for your deployed frontend application to be globally available through AWS CloudFront CDN.
+
 
-#### Building this tool
+### zero init
+
+The `zero init` command creates a new project and outputs an infrastructure configuration file with user input prompted responses into a file.  -> 📁 `YOUR_PROJECT_NAME/zero-project.yml`
+
+```shell
+# To create and customize a new project you run
+$ zero init
+
+## Sample project initilization
+✔ Project Name: zero-test
+🎉  Initializing project
+✔ EKS + Go + React
+✔ Should the created projects be checked into github automatically? (y/n): y
+✔ What's the root of the github org to create repositories in?: github.com/zero-test-org
+✔ Existing AWS Profiles 
+✔ default
+
+Github personal access token: used for creating repositories for your project
+Requires the following permissions: [repo::public_repo, admin::orgread:org]
+The token can be created at https://github.com/settings/tokens
+✔ Github Personal Access Token with access to the above organization: <MY_GITHUB_ORG_ACCESS_TOKEN>
+
+CircleCI api token: used for setting up CI/CD for your project
+The token can be created at https://app.circleci.com/settings/user/tokens
+✔ Circleci api key for CI/CD: <MY_CIRCLE_CI_ACCESS_TOKEN>
+✔ us-west-2
+✔ Production Root Host Name (e.g. mydomain.com) - this must be the root of the chosen domain, not a subdomain.: commitzero.com
+✔ Production Frontend Host Name (e.g. app.): app-zero-test-prod.
+✔ Production Backend Host Name (e.g. api.): api-zero-test-prod.
```

Comment:
i think its more clear for the readme to leave the frontend and backend as just `api.` and `app.`